CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Remember to bring your dependencies up to date with `./scripts/venvinstall.sh` w
 - Major: Potentially breaking, filters in commands are now applied even if the result of a substitution returns an error. (#1973)  
   This makes all `or_...` filters a lot more useful, but it may mean that some other filters will need some additional error handling.  
   This will require some experimentation, reporting errors in our GitHub issues for this is greatly appreciated.
+- Minor: Unhid the `Unfollow` Twitter button and disabled for users that are under level 1000. (#2104)
```

Comment:
I feel like this change doesn't add anything relevant so I'd prefer to not have a changelog for it at all. Minor is a great overstatement